x-pack/plugins/session_view/public/components/session_view/hooks.ts

```diff
@@ -17,8 +17,8 @@ export const useFetchSessionViewProcessEvents = (
   jumpToEvent: ProcessEvent | undefined
 ) => {
   const { http } = useKibana<CoreStart>().services;
-
-  const jumpToCursor = jumpToEvent && jumpToEvent['@timestamp'];
+  const [isJumpToBeginning, setIsJumpToBeginning] = useState<boolean>(false);
```

Comment:
what `isJumpToBeginning` means? jump to the first result or to the initial state?

Reply:
It should mean if the `jumpToEvent` is on the first page of the session, so that we can work around the infinite render with `query.data?.pages.length === 1` check in the `useEffect` hook below . I will rename it to `isJumpToFirstPage` to make it more clear

Reply:
Got it, yeah agree on updating the name